Dockerfile

```diff
@@ -105,13 +103,11 @@ RUN set -ex && apk --update --no-cache add \
     nodejs-npm
 
 RUN set -ex && apk --update --no-cache add \
-    protoc~=${proto_version} \
-    protobuf~=${proto_version} \
-    --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main
-
-RUN set -ex && apk --update --no-cache add \
-    grpc~=${grpc_version} \
-    grpc-cli~=${grpc_version} \
+    "protoc>${proto_version}" \
+    "protobuf~=${proto_version}" \
+    "grpc~=${grpc_version}" \
```

Comment:
Should this be consistent with prev command and use `>` ?

Reply:
oops

Reply:
fixed, thanks